frontend/src/components/commons/BackToTop.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from "react";
+import LOCALIZE from "../../text_resources";
+import scrollToTop from "../../helpers/scrollToTop";
+
+const styles = {
+  hiddenButton: {
+    visibility: "hidden"
+  },
+  displayedButton: {
+    display: "block",
+    position: "fixed",
+    bottom: "5%",
+    right: "3%"
+  }
+};
+
+class BackToTop extends Component {
+  state = {
+    displayButton: "hidden"
+  };
+
+  componentDidMount = () => {
+    window.addEventListener("scroll", this.handleOnScroll);
+  };
+
+  // handle onScroll event
+  handleOnScroll = () => {
+    const currentScroll = document.documentElement.scrollTop;
+    console.log(currentScroll);
+    if (currentScroll > 20) {
+      this.setState({ displayButton: "displayed" });
+    } else {
+      this.setState({ displayButton: "hidden" });
+    }
+  };
+
+  render() {
+    const { displayButton } = this.state;
+    return (
+      <div>
+        {displayButton === "displayed" && (
```

Comment:
Why do you need both hidden CSS and React to hide/show "display" and "hidden" elements? This seems redundant.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/87/commits/f9dd094e5d83a03971d4050f0b95ed4036af1101